frontend-ssr/redux/user/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface UserState {
+  info: {
+    name: 'string',
+    username: 'string',
+    email: 'string',
+    avatar: 'string',
+  }
+  notifications: Notifications;
+  page: number,
```

Comment:
`page` and `end` don't make sense to be on the `user` directly. They should be inside `notifications`, since they relate to that.

Reply:
yeah, as in the example with submissions.